src/planner/match/VertexIdSeek.cpp

```diff
@@ -35,86 +36,65 @@ bool VertexIdSeek::matchNode(NodeContext* nodeCtx) {
         return false;
     }
 
-    auto vidResult = extractVids(matchClauseCtx->where->filter.get());
-    if (!vidResult.ok()) {
+    if (node.alias == nullptr) {
+        // require one named node
```

Comment:
use node. anonymous instead, because node.alias is always not nullptr:
```
        if (alias == nullptr) {
            anonymous = true;
            alias = saveObject(new std::string(vctx_->anonVarGen()->getVar()));
        }
       ...
        nodeInfos[i].anonymous = anonymous;
        nodeInfos[i].alias = alias;
```

Reply:
Fixed.